src/main.py

```diff
@@ -0,0 +1,88 @@
+""" Main FastAPI app """
+import logging
+import os
+from pydantic import BaseModel, Field
+
+from typing import List
+
+from fastapi import FastAPI
+from fastapi.middleware.cors import CORSMiddleware
+from bayesian import run_one_1d_bayesian_optimization
+
+logger = logging.getLogger(__name__)
+
+version = "0.0.1"
+description = """
+"""
+app = FastAPI(
+    title="Materiom Active Learning API",
+    version=version,
+    description=description,
+    contact={
+        "name": "Materiom",
+    },
+)
+
+origins = os.getenv("ORIGINS", "*").split(",")
+app.add_middleware(
+    CORSMiddleware,
+    allow_origins=origins,
+    allow_credentials=True,
+    allow_methods=["*"],
+    allow_headers=["*"],
+)
+
+
+class OneData(BaseModel):
+    x: float = Field(..., description="x value")
+    y: float = Field(..., description="x value")
+
+
+class OneErrorBar(BaseModel):
+    mu: float = Field(..., description="Mean")
+    sigma: float = Field(..., description="standarad deviation")
+    x: float = Field(..., description="x value")
```

Comment:
What's the difference between x and mu? Could there be a little docstring for the OneErrorBar class?